spec/lib/folio_record_spec.rb

```diff
@@ -61,6 +61,65 @@
                                                                          ])
       end
     end
+
+    context 'when deriving from the instance record' do
+      let(:record) do
+        {
+          'instance' => {
+            'id' => '0e050e3f-b160-5f5d-9fdb-2d49305fbb0d',
+            'title' => 'The title',
+            'identifiers' => [
+              { 'value' => 'garbage' },
+              { 'value' => '(OCoLC-M)948533645' },
+              { 'value' => '2021236856' },
+              { 'value' => '2381-5868' },
+              { 'value' => '1485631076 (paperback)' },
+              { 'value' => '9781485631071 (paperback)' }
```

Comment:
it looked to me from the LOC docs that the extra info in here like ` (paperback) ` often ends up in subfield `q`. should we try to separate it that way, so that subfield `a` is only the numeric content?

Reply:
(question for @cbeer, I guess?)

Reply:
The indexer downstream is extracting the data:

https://github.com/sul-dlss/searchworks_traject_indexer/blob/main/lib/traject/config/sirsi_config.rb#L1656

So I'm not sure we care. If it we me, I wouldn't try too hard for these stub records 🤷‍♂️